src/data/faq.json

```diff
@@ -549,6 +549,14 @@
                                     "The EU specifications for booster vaccination certificates have changed. If your certificate no longer meets the current specifications, it is still valid, but it may not be recognized as a booster vaccination during an inspection. In such a case, you can be informed about it via the Corona-Warn-App version 2.20.3 and later with an appropriate message. You can then voluntarily request a new vaccination certificate free of charge via the app. After you have given your consent, the app transmits the vaccination certificate to the RKI for verification, which issues an updated certificate and transmits it to your app. You will then find the updated certificate in the certificate tab. The old certificate is automatically moved to the recycle bin and deleted after 30 days. The old certificate can be restored at any time within the 30 days.",
                                     "For more information, visit the blog: <a href='../../blog/2022-03-30-cwa-2-20/' target='_blank'>Corona-Warn-App Version 2.20</a>"
                                 ]
+                            },
+                            {
+                                "title": "I receive a notification that the validity of a certificate is about to expire. What do I have to do?",
+                                "anchor": "dcc_replacement_howto",
+                                "active": false,
+                                "textblock": [
+                                    "The technical validity of COVID digital certificates expires automatically after 365 days. For all relevant certificates, you will receive a notification about this in the Corona-Warn-App 28 days before expiration. In order to continue to be able to prove your vaccination status with the app, the relevant certificates must be updated. You will soon be able to carry out the necessary reissue yourself in the app with just a few clicks."

```

Comment:
I think in English you don't say that the validity expires, but rather the certificate expires. 

@MikeMcC399 what do you think?

Reply:
The sentence "The technical validity of COVID digital certificates expires automatically after 365 days." is fine and corresponds to the German text "Die technische Gültigkeit der digitalen COVID-Zertifikate läuft nach 365 Tagen automatisch ab."

Due to the fact that a certificate has two types of validities (see https://www.coronawarn.app/en/faq/results/#glossary_validity) it is better to explicitly mention the technical validity expiring, since this is different from an expiration of the effective validity.

Of course, if the technical validity expires, then the certificate as a whole also expires. I think this should be implicitly understood.

In any case I see no need to change the text.

Reply:
Thanks @MikeMcC399! I was unsure, but if you say it's fine that's good!

Reply:
@Ein-Tim @MikeMcC399 Thanks for review. We keep the wording for the moment, as suggested by @MikeMcC399 and also agreed with @GisoSchroederSAP. Still, if necessary we will adjust the wording.